source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,577 +3,765 @@
-      need to pass :option:`--format <FORMAT_NUMBER>`. You can find your
-      current format version by running:
+Mixer checks the validity of the bundle's syntax and name. Mixer also ensures
+the bundle can be parsed. Run this validation manually on bundle1 with the
+following command:
 
-      .. code-block:: bash
+.. code-block:: bash
 
-         # cat /usr/share/defaults/swupd/format
+   sudo mixer bundle validate bundle1
 
-Creating your next mix version
-==============================
+With the optional :option:`--strict` flag, the command additionally
+checks if the rest of the bundle header fields can be parsed, if they are
+non-empty, and if the bundle header ``Title`` field and the bundle filename
+match. Perform a strict validation of `bundle1` with the following command:
 
-**Update the next Mix version info**. To increment your mix version number for
-your next mix, run:
+.. code-block:: bash
 
-   .. code-block:: bash
+   sudo mixer bundle validate --strict bundle1
 
-      # mixer versions update
+Validate multiple bundles with the following command:
 
-This command will automatically update your mix version (stored in the 
-:file:`mixversion` file), incrementing it by 10. If you'd like to increment by a
-different amount, the :option:`--increment` flag can be used:
+.. code-block:: bash
 
-   .. code-block:: bash
+   sudo mixer bundle validate bundle1 bundle2 [bundle3 ...]
 
-      # mixer versions update --increment 100
+Managing bundles with Git
+=========================
 
-Alternatively, if you want set your mix version to a specific value, you can
-do so with the :option:`--mix-version` flag:
+If you initialized your workspace to be tracked as a Git repository
+with the :command:`mixer init --git` command, it might be useful to apply a git
+commit after modifying the mix bundle list or editing a bundle definition file.
 
-   .. code-block:: bash
+All the :command:`mixer bundle` commands on the previous sections support an
+optional :option:`--git` flag. The flag automatically applies a git commit
+when the command completes, for example:
+
+.. code-block:: bash
+
+   sudo mixer bundle remove --git bundle1
+
+Build the bundle chroots
+************************
+
+To build all the ``chroots`` based on the defined bundles, use the following
+command in your workspace:
+
+.. code-block:: bash
+
+   sudo mixer build chroots
+
+If the mix has many bundles, this step might take some time.
+
+By default, mixer uses the legacy chroot-builder. In this mode, mixer
+automatically gathers the bundle definition files for the bundles in the mix
+into a :file:`mix-bundles` directory. The directory's path is set in the
+`BUNDLE_DIR` variable in the :file:`builder.conf`. **Do not edit these
+files.** Mixer automatically clears out any contents in this directory before
+populating it on-the-fly as mixer builds the chroots.
+
+We have build a new chroot-builder into the mixer tool itself. While this is
```

Comment:
I changed the verb but incorporated is difficult to understand for non-native speakers. I replaced it instead with 'added' since it also avoids the confusion with build but is a simple word.